soc/hps_proto2_platform.py

```diff
@@ -113,7 +113,7 @@ def __init__(self, platform, sys_clk_freq):
 class Platform(LatticePlatform):
     # The NX-17 has a 450 MHz oscillator. Our system clock should be a divisor
     # of that.
-    clk_divisor = 7
+    clk_divisor = 9
```

Comment:
The commit message for this says "DO NOT MERGE" :-) but maybe you do want to merge this anyway? The fast slew rate should fix SPI flash corruption I guess, but maybe we still need 50MHz system clock to get this to route?

Reply:
I'll keep this for now, until the nextpnr slew rate fixes make it into CFU Playground.